tests/python/unittest/test_target_target.py

```diff
@@ -80,7 +80,31 @@ def test_target_create():
         assert tgt is not None
 
 
+def test_target_config():
+    """
+    Test that constructing a target from a dictionary works.
+    """
+    target_config = {
```

Comment:
Should also test map-type attributes.

Reply:
Can you expand on this a little? Do you mean add testing that confirms it fails when invalid attribute types are provided?

Reply:
I just meant we should also test the attribute which type is a map. Since one purpose of testing Python binding is to make sure we can correctly pass supported data structures to the C++ container. Specifically, the Python version of this test: https://github.com/apache/incubator-tvm/blob/master/tests/cpp/target_test.cc#L121

Reply:
Ah I see. It's a little strange to do this since none of supported options are maps. But I could add a test case that passes a map and then confirm it fails like the cpp version you linked.

Reply:
Added an analogous test in the latest commit. Let me know if this is what you were looking for.

Reply:
I saw the map test you added, but it is intented to fail so I'm not sure if it is sufficient.
@junrushao1994 could you comment?

Reply:
Just to be clear, there are no valid arguments that have type map. The tests you linked in `target_test.cc` also are intentionally failing.